app/portainer/views/stacks/create/createStackController.js

```diff
@@ -177,6 +179,13 @@ angular
       } catch (err) {
         Notifications.error('Failure', err, 'Unable to retrieve Custom Templates');
       }
+
+      try {
+        const endpoint = await EndpointService.endpoint(endpointId);
```

Comment:
no need to load the endpoint. we have a resolver in the router, which means it should available through dependency injection